### ISSUE TITLE
📎 HTML: Implement "Bracket same line" option

### ISSUE THREAD

**Issue by dyc3** [biomejs/biome#4024]
### Description

Add the option to `HtmlFormatterSettings` and implement the behavior.

See: https://prettier.io/docs/en/options.html#bracket-line

Before this can be started, this requires:
- #4017

**Comment by ematipico** [comment]
This option should be a top level option now, because we already have one for JSX

https://biomejs.dev/reference/configuration/#javascriptformatterbracketsameline

We could add the top level one, deprecate the JSX option and provide a migration. This can be done in a patch release.

Then, in v2.0 we will remove the deprecated option. What do you think?

**Comment by dyc3** [comment]
Makes sense to me, since this option will also eventually apply to all HTML ish languages.

**Comment by JonathonRP** [comment]
Any progress on this?

**Comment by ematipico** [comment]
@JonathonRP This is a technical issue; unless you want to help or have technical opinions, please refrain from asking "when an issue is ready", we're trying to avoid such comments. Something is ready when is ready.

**Comment by JonathonRP** [comment]
> @JonathonRP This is a technical issue; unless you want to help or have technical opinions, please refrain from asking "when an issue is ready", we're trying to avoid such comments. Something is ready when is ready.

Sorry, I had a use case. I'm using svelte and want the brackets on the same line. I was curious about the updates, I saw the one PR is merged but didn't see a PR for progressing this, I would be willing to help

**Comment by ematipico** [comment]
Yeah exactly, nobody claimed the issue yet. If you're interested in helping, you're more than welcome 😁 

You can use this issue to ask any questions, or even open a draft PR and start from asking questions there
